src/api/xdsl/rma/xdsl-rma.v6.service.js

```diff
@@ -0,0 +1,48 @@
+angular.module('ovh-api-services').service('OvhApiXdslRMAV6', ($resource, $cacheFactory) => {
+  const cache = $cacheFactory('OvhApiXdslRMAV6');
+  const queryCache = $cacheFactory('OvhApiXdslRMAV6Query');
+
+  const interceptor = {
+    response(response) {
+      cache.remove(response.config.url);
+      queryCache.removeAll();
+      return response.resource;
+    },
+  };
+
+  const resource = $resource('/xdsl/:xdslId/rma', {
+    xdslId: '@xdslId',
+    id: '@id',
+  }, {
+    query: {
+      method: 'GET',
+      isArray: true,
+      cache: queryCache,
+    },
+    get: {
+      method: 'GET',
+      cache,
+    },
+    getRMA: {
+      method: 'GET',
+      url: '/xdsl/:xdslId/rma/:id',
```

Comment:
no cache used ?

Reply:
nope, need fresh info

Reply:
Why not prefer using the `get` (L22) without the cache entry?